system/ui/lib/list_view.py

```diff
@@ -11,154 +9,266 @@
 from openpilot.system.ui.lib.toggle import Toggle, WIDTH as TOGGLE_WIDTH, HEIGHT as TOGGLE_HEIGHT
 from openpilot.system.ui.lib.widget import Widget
 
-ITEM_BASE_HEIGHT = 170
 LINE_PADDING = 40
-LINE_COLOR = rl.GRAY
+ITEM_BASE_HEIGHT = 170
 ITEM_PADDING = 20
-ITEM_SPACING = 80
 ITEM_TEXT_FONT_SIZE = 50
 ITEM_TEXT_COLOR = rl.WHITE
 ITEM_DESC_TEXT_COLOR = rl.Color(128, 128, 128, 255)
 ITEM_DESC_FONT_SIZE = 40
 ITEM_DESC_V_OFFSET = 140
-RIGHT_ITEM_PADDING = 20
 ICON_SIZE = 80
 BUTTON_WIDTH = 250
 BUTTON_HEIGHT = 100
-BUTTON_BORDER_RADIUS = 50
 BUTTON_FONT_SIZE = 35
-BUTTON_FONT_WEIGHT = FontWeight.MEDIUM
 
-TEXT_PADDING = 20
 
+# Type Aliases for Clarity
+StrSrc = str | Callable[[], str] | None
+BoolSrc = bool | Callable[[], bool]
 
-def _resolve_value(value, default=""):
+
+def _get_value(value, default=""):
   if callable(value):
     return value()
   return value if value is not None else default
```

Comment:
I have a typed version you can use here:

https://github.com/commaai/openpilot/pull/35543/commits/98bfd45712fa8e0625581a1599d9601b435db15a